index.html

```diff
@@ -0,0 +1,189 @@
+        <a href="#main-program">PROGRAM</a><br>
+        <a href="#featured-speakers">SPEAKERS</a><br>
+        <a href="#join">JOIN</a>
+      </nav>
+    </div>
+  </div>
+
+  <div class="flex-navbar container-fluid px-4">
+    <a class="navbar-brand col-" href="#home">
+      <img src="./images/Logo/small-logo.png" height="60px" alt="our logo / home">
+    </a>
+    <div class="menu-buttons">
+      <div class="flex-menu">
+        <a href="#main-program">Program</a>
+        <a href="#featured-speakers">Speakers</a>
+        <a href="#about">About</a>
+        <a href="#partners">Sponsors</a>
+      </div>
+    <button type= "button" class="btn btn-light border-danger rounded-0" id="navbar-join">Join SoA</button>
+    </div>
+  </div>
+  <!-- Home section --> 
+  <div id="home" class="container-fluid py-5">
+    <div class="container my-5 pt-5">
+      <h2>"Listen! music of the future past"</h2>
+      <h1>SOUND OF ANEMOIA </br> RETROWAVE GATHER </br>2023</h1>
+    </div>
+
+    <div class="container my-3 p-3 home-p">
+      <p class="m-0">Welcome to the annual gathering of synth enthusiasts! Join us for a celebration of the past, present, and future of electronic music as we explore the latest gear, techniques, and sounds.</p>
+    </div>
+
+    <div class="container">
+      <h3>2023.10.13(FRI) ~ 14(SAT)</h2>
+      <h4>@ Cyberdelia Complex, Berlin, Germany.</h3>
+    </div>
+  </div>
+
+  <!-- Main program --> 
+  <div class="container-fluid py-5" id="main-program">
+
+    <div class="section-title">
+      <h2>Main Program</h2>
+      <div class="limit-border-bottom"></div>
+    </div>
+
+    <div class="program-cards px-3">
+      <div class="program-item">
+        <img src="./images/icons/speaker-icon-small.png">
+        <h3>Lecture</h3>
+        <p>Explore the latest trends and techniques in synth music production with our lineup of expert speakers.</p>
+      </div>
+
+      <div class="program-item">
+        <img src="./images/icons/listen-icon-small.png">
+        <h3>Listen</h3>
+        <p>Immerse yourself in the world of electronic music with our cutting-edge sound gallery.</p>
+      </div>
+
+      <div class="program-item">
+        <img src="./images/icons/forum-icon.png">
+        <h3>Forum</h3>
+        <p>Share your ideas and connect with fellow enthusiasts in our open forum.</p>
+      </div>
+
+      <div class="program-item">
+        <img src="./images/icons/create.png">
+        <h3>Create</h3>
+        <p>Get hands-on with the latest hardware and software and bring your synth music visions to life.</p>
+      </div>
+
+      <div class="program-item">
+        <img src="./images/icons/network-icon-small.png">
+        <h3>Network</h3>
+        <p> Meet and collaborate with fellow synth music lovers from around the globe.</p>
+      </div>
+    </div>
+
+    <button id="join" class="join-btn">Join Sound of Anemoia 2023</button>
+    <a href="#" id="whole-program">SEE THE WHOLE PROGRAM</a>
```

Comment:
Please kindly adjust the button styling to match the given template. 
- Layout of the content
- Spacing `margin/padding`

Your design
![image](https://user-images.githubusercontent.com/78251927/233806766-3ee7a10c-27d6-473f-adb9-196df8f9ceb7.png)

Template
![image](https://user-images.githubusercontent.com/78251927/233806776-83652391-6254-441a-be41-66a1e7d22d7a.png)